src/pages/patientView/pathwayMapper/PatientViewPathwayMapper.tsx

```diff
@@ -141,20 +129,19 @@ export default class PatientViewPathwayMapper extends React.Component<
         });
         return keyed_genes;
     }
-    @computed get isNewStoreReady() {
+    @computed get isStoreReady() {
         return (
-            this.storeForAllData &&
-            this.storeForAllData.samples.isComplete &&
-            this.storeForAllData.mergedMutationData &&
-            this.storeForAllData.coverageInformation.isComplete &&
-            this.storeForAllData.mergedDiscreteCNADataFilteredByGene &&
-            this.storeForAllData
-                .mergedMutationDataIncludingUncalledFilteredByGene
+            this.patientStore &&
+            this.patientStore.samples.isComplete &&
+            this.patientStore.mergedMutationData &&
+            this.patientStore.coverageInformation.isComplete &&
+            this.patientStore.mergedDiscreteCNADataFilteredByGene &&
+            this.patientStore.mergedMutationDataIncludingUncalledFilteredByGene
         );
     }
     public render() {
         //control the data
-        if (this.isNewStoreReady) {
+        if (this.isStoreReady) {
             this.addGenomicData(this.alterationFrequencyData);
             this.getQueryGenes(this.alterationFrequencyData);
```

Comment:
This code segment is never executed, because `isStoreReady` is always false. It appears to me that we don't need to call `addGenomicData` at all for patient view page. We should probably remove this segment together with the `isStoreReady` function.